src/webviews/extension-side/webviews/webview.node.ts

```diff
@@ -37,7 +37,11 @@ export abstract class Webview implements IWebview {
     ) {
         const webViewOptions: WebviewOptions = {
             enableScripts: true,
-            localResourceRoots: [Uri.file(this.options.rootPath), Uri.file(this.options.cwd), ...additionalRootPaths]
+            localResourceRoots: [
+                fs.normalize(this.options.rootPath),
```

Comment:
Why do you need to normalize here? It should load regardless.

Reply:
Wouldn't normalize only be necessary for comparison?

Reply:
`file://` URls do not work well in the web. I've had to replace all the code that uses `Uri.file` in order to make things work in the web.

Reply:
I think we can follow how VS Code builtin webview extensions handle `localResourceRoots` with `Uri` https://github.com/microsoft/vscode/blob/main/extensions/markdown-language-features/src/preview/preview.ts#L430-L444

Reply:
> file:// URls do not work well i

i think the root problem is the  that were still not using uris in pokeberry off places. i think we should be using uris instead of assuming that files paths have the scheme such as `file://`..

ie if we have pieces that hard code Uri.file, then that's a problem, i think it should be uri.join asking with the real paths from either the work space or the extension root using context.extensionUri.

what im getting at is, we shouldn't have to hardcode Uri.file nor should we have to parse using Uri.parse

Reply:
> use url path joins where possible.

that's my point too